src/pages/kwonojae/Login/Login.scss

```
@@ -0,0 +1,60 @@
+@import '../../../styles/Variable.scss';
+
+.loginOJ {
+  @extend .flexCS;
+  align-items: center;
+  width: 350px;
+  height: 380px;
+  margin: 150px auto 0px auto;
+  border: $border1;
+  border-radius: 1px;
+  h1 {
+    margin: 32px 0px 0px 0px;
+    text-align: center;
+    font-family: 'Lobster', cursive;
+    font-size: 41px;
+  }
+  input {
+    width: 268px;
+    height: 38px;
+    padding: 9px 0px 7px 8px;
+    background-color: $inputBgCol1;
+    border: $border1;
+    border-radius: 3px;
+    &::placeholder {
+      color: #8c8c8c;
+    }
+  }
+  .inputID {
+    margin: 36px 0px 0px 0px;
+  }
+  .inputPW {
+    margin: 6px 0px 0px 0px;
+  }
+  a {
+    cursor: default;
+    .loginBtn {
+      margin-top: 14px;
+      width: 268px;
+      height: 30px;
+      color: $white;
+      border: none;
+      border-radius: 4px;
+      font-size: 14px;
+      font-weight: 600;
```

```suggestion
      font-weight: 700;
```
font-weight 주실때 400, 700이 주로 사용된다고 합니다~! 아래 참고하시면 좋을거같아요?(MDN 출처)

- normal
   - 보통 폰트 가중치. 400 과 같음.
- bold
   - 굵은 폰트 가중치. 700 과 같음.

---

👍

---

수정했습니다